Makefile

```diff
@@ -164,14 +180,19 @@ MACHINE_SIMPLE-$1=$$(MACHINE_SIMPLE-$$(ARCH-$1))
 else
 MACHINE_SIMPLE-$1=$$(ARCH-$1)
 endif
-SDK-$1=$$(basename $1)
+
+MACOSX_DEPLOYMENT_TARGET=10.6
```

Comment:
The python binary produced with `MACOSX_DEPLOYMENT_TARGET=10.8` also runs on Mountain Lion (10.8.5), verified in a VM.